core/src/impl/Kokkos_BitOps.hpp

```diff
@@ -95,16 +95,17 @@ inline int int_log2_host(unsigned i) {
 #endif
 }
 
-#if defined(__EDG__)
-#pragma push
+#if defined(__EDG__) && !defined(KOKKOS_COMPILER_INTEL)
+#pragma push && !defined(KOKKOS_COMPILER_INTEL)
```

Comment:
is this copy paste error?

Reply:
@crtrott clang-format error, not sure how to prevent it without breaking the clang-format check, any suggestions?

Reply:
Fixed now